### PR TITLE
Add dnsConfig to head, worker and additional workers

### PR DESCRIPTION
**Pull request by edward2a**
## Why are these changes needed?

To allow customisation of the dnsConfig for ray pods.

## Related issue number

N/A

## Checks

- [ ] I've made sure the tests are passing.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [X] This PR is not tested :(

### CODE REVIEW

**Comment by kevin85421** [comment]
@MortalHappiness can you review this PR?

**Comment by MortalHappiness** [comment]
@edward2a could you add comments for these variables to `helm-chart/ray-cluster/values.yaml`?

**Comment by kevin85421** [comment]
@edward2a any updates?

**Comment by edward2a** [comment]
> @edward2a any updates?

@MortalHappiness apologies for the delay, time is limted.
Now updated, let me know for any further changes required.

File: helm-chart/ray-cluster/values.yaml [hunk 1]
```diff
@@ -116,6 +116,18 @@ head:
...
+    # See https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config
+    # dnsConfig:
+    #   nameservers:
+    #     - 8.8.8.8
+    #   searches:
+    #     - example.local
+    #   options:
+    #     - name: ndots
+    #       value: "2"
+    #     - name: edns0
```
Comment (MortalHappiness) [inline helm-chart/ray-cluster/values.yaml:129]: Unindent this once. This should be at the same indentation level with `headService`.

**Review by MortalHappiness — APPROVED** [review]
LGTM

**Pull request merged**